Letterbook.Adapter.ActivityPub/Mappers/AsApMapper.cs

```diff
@@ -12,6 +14,32 @@ public static class AsApMapper
         ConfigureNote(cfg);
     });
 
+    public static MapperConfiguration ActorConfig = new(cfg =>
+    {
+        ConfigureActor(cfg);
+    });
+
+    private static void ConfigureActor(IMapperConfigurationExpression cfg)
+    {
+        cfg.CreateMap<Models.Profile, PersonActorExtension>()
+            .ForMember(dest => dest.Id, opt => opt.MapFrom(src => src.Id))
+            .ForMember(dest => dest.Summary, opt => opt.MapFrom(src => src.Description))
+            .ForMember(dest => dest.Name, opt => opt.MapFrom(src => src.DisplayName))
+            .ForMember(dest => dest.PreferredUsername, opt => opt.MapFrom(src => src.Handle))
+            .ForMember(dest => dest.Attachment, opt => opt.Ignore())
+            .ForMember(dest => dest.Liked, opt => opt.Ignore())
+            .ForMember(dest => dest.PublicKey,
+                opt => opt.MapFrom<SigningKeyConverter, IList<SigningKey>>(src => src.Keys))
+            .ForMember(dest => dest.Endpoints, opt => opt.Ignore())
```

Comment:
Are endpoints intentionally ignored?

Reply:
This is mostly copied from the letterbook.ap mapper, and I hadn't implemented endpoints. So, probably not.